admin_guide/manage_scc.adoc

```diff
@@ -0,0 +1,153 @@
+= Managing Security Context Constraints
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+You can manage
+link:../architecture/additional_concepts/authorization.html#security-context-constraints[security context constraints]
+in your instance via the `oc` command as a normal API objects.
+
+When you perform node management operations, the CLI interacts with
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node-object-specification[node
+objects] that are representations of actual node hosts. The
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#master[master]
+uses the information from node objects to validate nodes with
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[health
+checks].
+
+== Listing Security Context Constraints
+
+[options="nowrap"]
+----
+$ oc get scc
+NAME         PRIV      CAPS      HOSTDIR   SELINUX     RUNASUSER
+privileged   true      []        true      RunAsAny    RunAsAny
+restricted   false     []        false     MustRunAs   MustRunAsRange
+----
+
+== Creating New Security Context Constraints
+
+Creating a new SCC is accomplished with the `oc create` command.  First, define the SCC in a json
+or yaml file and then run `oc create` passing the file.
+
+[options="nowrap"]
+----
+{
+  "kind": "SecurityContextConstraints",
+  "apiVersion":"v1beta3",
+
+  "metadata": {
+    "name": "scc-admin",
+    "labels": {
+            "name": "scc-admin"
+    }
+  },
+  "allowPrivilegedContainer": true,
+  "runAsUser": {
+    "type": "RunAsAny"
+  },
+  "seLinuxContext": {
+    "type": "RunAsAny"
+  },
+  "groups": ["system:serviceaccounts"]
+}
```

Comment:
Also, if it's all the same, think we're trying to use YAML for examples in docs since it's friendlier to read.